src/compiler/crystal/types.cr

```diff
@@ -1182,6 +1182,20 @@ module Crystal
       false
     end
 
+    protected def define_allocate(metaclass)
+      if metaclass.instance_type.abstract?
+        metaclass.add_def(
+          Def.new("allocate",
+            body: Call.new(nil, "raise",
+              args: [StringLiteral.new("can't instantiate abstract class #{metaclass.instance_type}")] of ASTNode,
```

Comment:
```suggestion
              args: [StringLiteral.new("Can't instantiate abstract class #{metaclass.instance_type}")] of ASTNode,
```

IIRC runtime errors are written in _Title case_.

Reply:
No, compiler errors are lowercase.